java-squid/src/main/java/org/sonar/java/ast/parser/TreeFactory.java

```diff
@@ -1818,6 +1833,7 @@ public NewArrayTreeImpl newArrayCreatorWithDimension(AstNode openBracketToken, E
     }
 
     List<AstNode> children = Lists.newArrayList();
+    // TODO SONARJAVA-547 brackets should be stored
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)